cmd/cloudshell_open/main.go

```diff
@@ -242,6 +238,20 @@ func run(opts runOpts) error {
 
 	image := fmt.Sprintf("gcr.io/%s/%s", project, serviceName)
 
+	existingEnvVars := map[string]struct{}{}
+	_, err = describe(project, serviceName, region)
+	if err == nil {
```

Comment:
we're not checking actual error here.
so it seems like we actually need something along the lines of checking service exists `(bool, error)` instead of just ignoring `err` here.

this is hacky currently, but it'll work, but it might mask other issues in the future.

Reply:
Yeah, the error could be anything.  We don't have a good way when wrapping the `gcloud` command to parse actual errors.  But we could use the API directly.  I'll add a todo for now.